roles/splunk_standalone/molecule/default/playbook.yml

```diff
@@ -53,7 +53,7 @@
       http_enableSSL_cert: null
       http_enableSSL_privKey: null
       http_enableSSL_privKey_password: null
-      http_port: 8000
+      http_port: 8080
```

Comment:
In general this looks fine to me, but I would like a short comment here about why this changed to 8080.  Mostly so that lazy engineers like me don't accidentally revert it.

Reply:
I've changed it to test the custom port scenario. I'm sorry, a bit new at this. Should I add that as a comment in the code and check back in?

Reply:
The comment in the code.  What you have right now is perfect. Thanks!